components/Blog/blogList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const Blog = ({ content }) =>
+  (<div className='blog-component pa4' >
+    <img src={content.imgUrl} className='mt5 mr4 mb4 ml0 w4 h4 br-100' alt={content.school} />
+    <h2 className='title tl pa3 ttc f3 measure tf-lato'>{content.title}</h2>
```

Comment:
wrap the h2, div, and a in a div tag so we can use flex to have the content and img side by side

Reply:
the flex will be added to the parent `blog-component` `div`